daq/forchestrator.py

```diff
@@ -72,7 +72,7 @@ def get_overview(self, path, params):
 
     def get_switches(self, path, params):
         """Get the state of the switches"""
-        return self._faucet_states_collector.get_switches()
+        return self._faucet_states_collector.get_switches(path[1:])
```

Comment:
should be a query parameter for this, and passed in through params:

e.g., /switches?name=t1sw1